README.rst

```diff
@@ -52,6 +55,11 @@ After installation, try the following examples that demonstrate how to visualize
 - `NumPy array images <https://github.com/InsightSoftwareConsortium/itk-jupyter-widgets/blob/master/examples/NumPyArray.ipynb>`_ (processed with SciPy)
 - `VTK vtkImageData <https://github.com/InsightSoftwareConsortium/itk-jupyter-widgets/blob/master/examples/vtkImageData.ipynb>`_
 
+Examples on Binder
+--------
+
+-  `3D ITK Images <https://mybinder.org/v2/gh/4QuantOSS/itk-jupyter-widgets/master?filepath=examples%2F3DImage.ipynb>`_
```

Comment:
`4QuantOSS` -> `InsightSoftwareConsortium`, see above.

Reply:
yes, I had just left it on mine since otherwise the links don't work